docs/docs/tascomi-ingestion.md

```diff
@@ -0,0 +1,73 @@
+
+- An initial full ingestion from Tascomi API (only once, happened thougout October 2021)
+- A daily call to the Tascomi API to get latest updated records (increment)
+- Parsing of the json data increment returned by the API
+- Refinement of the parsed data to recast all columns to the right data type
+- Creation of a full snapshot by applying the daily increment to the previous snapshot
+
+# Details of individual steps
+## 0 - Initial full ingestion
+This [process](https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/stg%20tascomi_api_ingestion_planning/script) queries one API endpoint (e.g. the applications endpoint) and writes the data into a table of the same name.
+
+This process wrote into the raw zone, in the 'api_response' bucket. The data is partitioned by import_date.
+## 1 - Daily ingestion of latest updated records
+This [process](https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/stg%20tascomi_api_ingestion_planning/script) is the same as the previous one, but it only queries the data updated since a certain date. It relies on the ```last_updated``` column that is present on all Tascomi tables.
+
+This process writes into the raw zone, in the 'api_response' bucket. The data is partitioned by import_date.
+## 2 - Daily parsing of the json increments
+This [process](https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/stg%20tascomi_parse_tables_increments_planning/script) uses job bookmarking to only process new increments.
+It processes all tables in a loop. For each table, the large json blob containing all the fields is exploded into separate textual columns.
+
+This process writes into the raw zone, in the 'parsed' bucket. The data is partitioned by import_date.
+## 3 - Daily refinement of the parsed increments
+This [process](https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/stg%20tascomi_recast_tables_increments_planning/script) uses job bookmarking to only process new increments.
+It processes all tables in a loop. For each table, the text columns are converted into correct data types (dates, boolean etc.). It uses a [column type dictionary](https://github.com/LBHackney-IT/Data-Platform/blob/main/scripts/jobs/planning/tascomi-column-type-dictionary.json) saved in a separate json file. This dictionary was created semi-automatically with FME (an ETL tool used in the Data and Insight team), by converting the list of columns described in the [API endpoints documentation](https://hackney-planning.tascomi.com/rest/v1/documentation.html?public_key=dd95bcd473f46a4325a4021d54500c7d#available-resources).
+
+This process writes into the refined zone, in the 'increment' bucket. The data is partitioned by import_date.
+## 4 - Creation of the daily snapshot
```

Comment:
```suggestion
### 0 - Initial full ingestion
This [process](https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/stg%20tascomi_api_ingestion_planning/script) queries one API endpoint (e.g. the applications endpoint) and writes the data into a table of the same name.

This process wrote into the raw zone, in the 'api_response' bucket. The data is partitioned by import_date.
### 1 - Daily ingestion of latest updated records
This [process](https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/stg%20tascomi_api_ingestion_planning/script) is the same as the previous one, but it only queries the data updated since a certain date. It relies on the ```last_updated``` column that is present on all Tascomi tables.

This process writes into the raw zone, in the 'api_response' bucket. The data is partitioned by import_date.
### 2 - Daily parsing of the json increments
This [process](https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/stg%20tascomi_parse_tables_increments_planning/script) uses job bookmarking to only process new increments.
It processes all tables in a loop. For each table, the large json blob containing all the fields is exploded into separate textual columns.

This process writes into the raw zone, in the 'parsed' bucket. The data is partitioned by import_date.
### 3 - Daily refinement of the parsed increments
This [process](https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/stg%20tascomi_recast_tables_increments_planning/script) uses job bookmarking to only process new increments.
It processes all tables in a loop. For each table, the text columns are converted into correct data types (dates, boolean etc.). It uses a [column type dictionary](https://github.com/LBHackney-IT/Data-Platform/blob/main/scripts/jobs/planning/tascomi-column-type-dictionary.json) saved in a separate json file. This dictionary was created semi-automatically with FME (an ETL tool used in the Data and Insight team), by converting the list of columns described in the [API endpoints documentation](https://hackney-planning.tascomi.com/rest/v1/documentation.html?public_key=dd95bcd473f46a4325a4021d54500c7d#available-resources).

This process writes into the refined zone, in the 'increment' bucket. The data is partitioned by import_date.
### 4 - Creation of the daily snapshot
```